### PR TITLE
Remove references to `test_procedure`

### PR DESCRIPTION
**Pull request by tchak**

### CODE REVIEW

File: app/models/procedure_path.rb [hunk 1]
```diff
@@ -3,32 +3,18 @@ class ProcedurePath < ApplicationRecord
...
-    end
-    if test_procedure == new_procedure
-      update(test_procedure: nil)
-    end
-    if procedure.nil? && test_procedure.nil?
-      destroy
-    end
+  def hide!
+    update(procedure: nil)
+    destroy
```
Comment (kemenaran) [inline app/models/procedure_path.rb]: `update!` et `destroy!`, non ?

File: app/models/procedure_path.rb [hunk 2]
```diff
@@ -3,32 +3,18 @@ class ProcedurePath < ApplicationRecord
...
   def publish!(new_procedure)
-    if procedure != new_procedure
-      if procedure&.publiee?
-        procedure.archive!
-      end
-      update(procedure: new_procedure)
+    if procedure&.publiee?
+      procedure.archive!
     end
+    update(procedure: new_procedure)
```
Comment (kemenaran) [inline app/models/procedure_path.rb]: `update!` ?

Comment (tchak) [reply app/models/procedure_path.rb]: done

File: app/models/procedure_path.rb (hunk 1 above)
Comment (tchak) [reply app/models/procedure_path.rb]: done

**Pull request merged**